dashboards/ephemeral-ns-operator-dashboard.yaml

```diff
@@ -647,7 +646,7 @@ data:
             {
               "datasource": {
                 "type": "prometheus",
-                "uid": "${datasource}"
+                "uid": "PA6124D83F67FD625"
```

Comment:
Should these be fixed data source IDs instead of `${datasource}`?

Reply:
Got an answer from @maknop - this is all good

Reply:
They were set to `${datasource}` so I changed them to be set to the specific datasources required for each panel.